pvc_to_pvs_image_import/_topics/preparing_powervc_node.md

```diff
@@ -0,0 +1,223 @@
+### Assumptions:
+
+1. You have the latest installation of the ManageIQ base `>= 98cb43d08130ae...`.
```

Comment:
Instead of `ManageIQ` you should use `{{ site.data.product.title_short }}` to make it also render for downstream documentation.

Example:
- https://github.com/ManageIQ/manageiq-documentation/blame/master/managing_providers/_topics/adding_ibm_cloud_powervs_providers.md#L4
- https://www.manageiq.org/docs/reference/latest/managing_providers/index.html#adding-ibm-power-systems-virtual-servers-providers